AppTaskQueue/appscale/taskqueue/queue.py

```diff
@@ -0,0 +1,160 @@
+import re
+
+# A regex rule for validating queue names.
+QUEUE_NAME_RE = re.compile('^[a-zA-Z0-9-]{1,100}$')
+
+# A regex rule for validating push queue rate.
+RATE_REGEX = re.compile(r'^(0|[0-9]+(\.[0-9]*)?/[smhd])')
+
+# A regex rule for validating push queue age limit.
+AGE_LIMIT_REGEX = re.compile(r'^([0-9]+(\.[0-9]*(e-?[0-9]+))?[smhd])')
+
+# The default rate for push queues.
+DEFAULT_RATE = '5/s'
+
+# The default number of task retries for a queue.
+DEFAULT_RETRY_LIMIT = None
+
+# The queue default time limit for retrying a failed push task.
+DEFAULT_AGE_LIMIT = None
```

Comment:
I think 30 days is for the max countdown/ETA (ie how long to wait before executing the task). This is for the expiration. I couldn't find a max default age limit in the docs, but here's [an answer](http://stackoverflow.com/questions/23306301/what-are-the-defaults-for-a-task-queue-in-appengine) on stack overflow that suggests it's `None` for the queue default.